core/src/main/java/haveno/core/api/XmrConnectionService.java

```diff
@@ -596,7 +601,7 @@ public void onConnectionChanged(MoneroRpcConnection connection) {
             } else if (!isInitialized) {
 
                 // set connection from startup argument if given
-                connectionManager.setAutoSwitch(false);
+                connectionManager.setAutoSwitch(true);
```

Comment:
This functionality should not be changed in this PR, which only adds the ability to query auto switch.